packages/grafana-ui/src/components/Layout/Box/Box.tsx

```diff
@@ -14,7 +14,7 @@ export type BorderColor = keyof GrafanaTheme2['colors']['border'] | 'error' | 's
 export type BorderRadius = keyof ThemeShape['radius'];
 export type BoxShadow = keyof ThemeShadows;
 
-interface BoxProps {
+interface BoxProps extends Omit<React.HTMLAttributes<HTMLElement>, 'className' | 'style'> {
```

Comment:
I think you still need to forward the props to the div for this to work.

Reply:
Yes very true 😅